Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -228,6 +237,41 @@ def revoke_user_session(self, user):
             resp_type="text"
         )
 
+    def get_signins(self, user: str, ago: str, ofilter: str, page_size: int, page_url: str):
+        if page_url:
+            response = self.ms_client.http_request(method='GET', url_suffix='auditLogs/signIns', full_url=page_url)
+        else:
+            filters = f"$filter=createdDateTime ge {parse_signins_timestamp(ago)}"
+            if user:
+                filters += f" and userPrincipalName eq '{user}'"
+            if ofilter:
+                filters += f" and {ofilter}"
+            response = self.ms_client.http_request(method='GET', url_suffix=f"auditLogs/signIns?{filters}&$top={page_size}")
+        next_page_url = response.get('@odata.nextLink')
+        signins = response.get('value')
+        return signins, next_page_url
+
+    def force_user_password_change(self, user: str, revoke_sessions: bool):
+        body = {
+            "passwordProfile":
+                {
+                    "forceChangePasswordNextSignInWithMfa": True
+                }
+        }
+
+        if revoke_sessions:
+            self.ms_client.http_request(
+                method='POST',
+                url_suffix=f'users/{quote(user)}/revokeSignInSessions',
+                resp_type="text"
+            )
+
+        self.ms_client.http_request(
```

Comment:
Does this call which forces the password change, not revoke the sessions as well?
Also if the flow first revokes the sessions and then forces password change, shouldn't the `revoke_sessions` argument will always be true? 
What is the use case if the user enters false?